lib/hammer_cli_katello/repository.rb

```diff
@@ -255,6 +262,27 @@ def import_upload_ids(ids)
         }
         resource.call(:import_uploads, params)
       end
+
+      def progress_message(filename)
+        _('Uploading %{filename}') % {:filename => filename}
+      end
+
+      def render_result
+        print_message _('finished')
```

Comment:
But you didn't introduce this new code in other areas? If there are similar areas, then I agree make changes in a different PR. This one is a nice small atomic change, that what I am asking about should take 15 minutes to verify and adjust if need be.